src/test/javascript/portal/details/LayerControlPanelSpec.js

```diff
@@ -121,19 +121,17 @@ describe("Portal.details.LayerControlPanel", function() {
             layerControlPanel._visibilityButtonChecked(null, true);
 
             expect(window.trackLayerControlUsage).toHaveBeenCalledWith(
-                'layerControlTrackingActionVisibility',
-                'on',
-                'Data Collection Title'
+                'visibility (overlay)',
+                'on - Data Collection Title'
```

Comment:
Should this be in a different format to the other trackLayerControlUsage events (and different to what we've recorded previously)?

Reply:
Yes after a discussion with @anguss00 we captured for the `visibility` action the amount of times that "on" or "off" was selected. Really what we need now is to track when a user changes the visibility of a layer, what sort of layer it was therefore what part of the GUI was used.
So now we will have three new actions for visibility - baselayer/datalayer/overlay, and then for the label prefix the action state (on or off) with the layername. So in the case of the new datalayer we can see which datalayer was selected. 
Additionally this pull request only tracks that a baselayer was turned on, not that another baselayer was turned off.

Reply:
Yes, but I don't understand why we have to bundle the collection title in the label parameter rather than using the separate collection parameter.